app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def show
+    puts params
+  end
+end
```

Comment:
- [ ] Good job so far, but you're expected to have an index and show methods in this controller. Kindly add the index method.

```ruby
 def index; end

  def show; end
```

Reply:
well resolved! ✔️